plugins/MultiSites/API.php

```diff
@@ -313,10 +314,11 @@ private function buildDataTable($idSites, $period, $date, $segment, $_restrictSi
 
             $pastArchive = Archive::build($idSites, $period, $strLastDate, $segment, $_restrictSitesToLogin);
             $pastData = $pastArchive->getDataTableFromNumericAndMergeChildren($fieldsToGet);
-
+            $pastData->filter('ReplaceColumnNames', array($columnNameRewrites));
```

Comment:
To be honest: I haven't done much with the ProxySite plugin yet. 
Filtering the $pastData array here ensures that after the call the columns are already renamed. This fixes the broken metric names.

I had a quick look at the ProxySite plugin. It does some special handling for those broken metrics, which we might need to remove to make it work again maybe.

In any case I guess it would make problems if a proxy site of an older version is included, as it e.g. also might not serve the hits metric. But as that plugin is private we can maybe ignore that as long as we ensure that the proxied instances are updated around the same time.

Reply:
Also had a quick look at it, seems like the column renaming is done in those cases as expected. At least the combination of `5.2.0-b5` and this branch looks like it works just fine.